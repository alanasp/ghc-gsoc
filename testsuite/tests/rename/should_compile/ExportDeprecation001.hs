{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import ExportDeprecation001_A

foo = func1
