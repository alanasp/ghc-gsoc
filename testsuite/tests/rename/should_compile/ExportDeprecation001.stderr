
ExportDeprecation001.hs:9:7: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘func1’ (imported from ExportDeprecation001_A):
    Deprecated: "func1 export is deprecated"
